fdbserver/worker.actor.cpp

```diff
@@ -2111,6 +2151,7 @@ ACTOR Future<Void> fdbd(Reference<ClusterConnectionFile> connFile,
 
 const Role Role::WORKER("Worker", "WK", false);
 const Role Role::STORAGE_SERVER("StorageServer", "SS");
+const Role Role::TESTING_STORAGE_SERVER("TestingStorageServer", "TSS");
```

Comment:
FYI @sfc-gh-etschannen @sfc-gh-mpilman this doesn't work :(
Any suggestions on a better 2-character abbreviation? I could do "ST" for "storage test"?

Assertion abbreviation.size() == 2 failed @ /root/src/fdbserver/WorkerInterface.actor.h 801:
  addr2line -e fdbserver.debug -p -C -f -i 0x27e6f84 0x17180a3 0x87e01c 0x2be28ad

Reply:
I take it we ruled out changing what tester is?

Reply:
I originally did that but we decided it was better not to

Reply:
Can we change the above assertion? Do we know whether there's any tooling relying on that? I am fine with `ST` but imho `TSS` would be better (as this is what everybody calls it).